lua/gitlinker/git.lua

```diff
@@ -1,97 +1,64 @@
 local M = {}
 
 local job = require("plenary.job")
+local path = require("plenary.path")
 
-local function get_remotes()
-  local remotes
-  local p = job:new({ command = "git", args = { "remote" } })
+-- wrap the git command to do the rigth thing alwasy
+-- @args: can be any git subcommand
+local function git(args)
```

Comment:
This comment is spurious and can be removed, I think the parameter `args` in a function called `git` is very self-explanatory.

Besides, there is no need for formal docs in local non-public functions!